### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by loic-sharma**
cc @robert-ancell

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,33 +27,52 @@ The demos in this repository run on the following platforms:
...
+   git remote add dkwingsmt https://github.com/dkwingsmt/engine/
+   git fetch dkwingsmt fffa6e19f5b71bedd55ac3a7fa6acfd85eda5983
+   git checkout fffa6e19f5b71bedd55ac3a7fa6acfd85eda5983
+   ```
 
-6. In **this repo**, update packages.
-```
-flutter pub get
-```
+4. macOS only: in **this repo**, edit pubspec.yaml, and add the following dependency override to use the custom `dart:ui` library:
```
Comment (dkwingsmt) [inline README.md]: Why does macOS need to do this?

Comment (loic-sharma) [reply README.md]: Ah good point, that shouldn't be necessary any more.

Comment (loic-sharma) [reply README.md]: Removed!

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**